tests/operator_test.go

```diff
@@ -2551,6 +2551,8 @@ spec:
 				panic(err)
 			}
 
+			By("Looking for pods with " + productComponent + " component")
```

Comment:
@ksimon1 Now I see why this failed. We are hardcoding the label in 53 releases. Can you take this?